csharp_test/code_20181216/IndividualPriceView.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Windows.Forms;
+
+namespace Distribution.View
+{
+    public partial class IndividualPriceView : Form
+    {
+        private int number;
```

Comment:
number is bad name. Put specific purpose for that number
such as checkedItemNum